integrations/csm-notification-service/internal/dispatch/dispatch.go

```
@@ -412,10 +416,15 @@ func (d *Dispatcher) handleCaseCreated(ctx context.Context, record eventbus.Reco
-		} else if chatErr := d.googleChat.SendIncidentAlert(ctx, product, p.CaseTitle, p.Description, d.links.CSMLink(p.CaseID)); chatErr != nil {
-			errs = append(errs, chatErr)
-			d.forget(chatKey)
-			chatOwned = false
+		} else {
+			severityLabel, severityColor := severityLabelAndColor(p.Priority)
+			caseLink := d.links.CSMLink(p.CaseID)
+			title := truncateTitle(p.CaseTitle, maxChatTitleLength)
+			if chatErr := d.googleChat.SendCaseCreatedAlert(ctx, product, severityLabel, severityColor, displayCaseRef(p.CaseNumber, p.CaseID), p.WSO2CaseID, p.Product, title, caseLink); chatErr != nil {
+				errs = append(errs, chatErr)
+				d.forget(chatKey)
+				chatOwned = false
```

golangci-lint (2.12.2)

[error] ineffectual assignment to chatOwned

(ineffassign)